flowzone.yml

```diff
@@ -2268,6 +2270,19 @@ jobs:
 
           result=$(echo "${newjson}" | jq '. | @json' )
           echo "result=${result}" >> $GITHUB_OUTPUT
+
+      # Disable auto-merge on any existing PRs if the current repo required reviewer count is
+      # greater than the required reviewer count being applied by Flowzone, likely a one-time change
+      # This should be done BEFORE applying the new rules
+      - name: Disable PR auto-merge
+        if: |
+          fromJSON(steps.parse_prepare_protection_rules.outputs.old_required_approving_review_count) > fromJSON(inputs.required_approving_review_count)
+        run: |
+          for pr in $(gh pr list --state=open --draft=false --json=number | jq -r '.[].number')
```

Comment:
Should this apply to drafts too? I can se someone being surprised when the PR they took out of draft immediately merges thinking it would need a review.

Reply:
I had the same thought, but as far as I can tell a PR in a draft state cannot have the automerge flag enabled. I also confirmed that marking as Draft removes the existing setting.

We could remove the `--draft=false` filter though, since we are ignoring errors on this line. It might just mean extra processing that isn't required.

Reply:
>  I also confirmed that marking as Draft removes the existing setting.

Oh, then that prevents the scenario I was describing. I think it's ok to leave this way then